tests/unittests/test_net.py

```diff
@@ -99,7 +99,7 @@
     'type': 'physical',
     'subnets': [{'broadcast': '10.0.0.255', 'control': 'manual',
                  'gateway': '10.0.0.1',
-                 'dns_search': ['foo.com'], 'type': 'static',
+                 'dns_search': ['foo.com'], 'type': 'none',
```

Comment:
I don't think this is correct; per https://cloudinit.readthedocs.io/en/latest/topics/network-config-format-v1.html#subnet-ip I think this should still be:

```suggestion
                 'dns_search': ['foo.com'], 'type': 'static',
```

Reply:
In that case code is buggy somewhere, and I must add the translation from 'none' to 'static' somewhere.

Reply:
Yep, presumably we're just reusing the value somewhere as they happened to be the same before this change.